ui/app/AppLayouts/Onboarding/views/LoginView.qml

```diff
@@ -48,7 +48,7 @@ Item {
     }
 
     Component.onCompleted: {
-        resetLogin()
+        doLogin("111111")
```

Comment:
missed to remove? and I know your fav password now :D lol

Reply:
😱